src/promptflow/promptflow/_utils/load_data.py

```diff
@@ -9,12 +9,14 @@
 
 import pandas as pd
 
+from promptflow._constants import PROMPT_FLOW_DIR_NAME
+from promptflow._utils.multimedia_utils import resolve_multimedia_data_recursively
 from promptflow.exceptions import ErrorTarget, UserErrorException
 
 module_logger = logging.getLogger(__name__)
 
 
-def _pd_read_file(local_path: str, logger: logging.Logger = None) -> pd.DataFrame:
+def _pd_read_file(local_path: str, logger: logging.Logger = None, *, enable_parse_image_path=False) -> pd.DataFrame:
```

Comment:
Why add a parameter here? In which case will this be set to True?

Reply:
SDK/CLI also uses the functions in this file. So I add this param and set it True in batch engine to prevent breaking change.